tests/ui/nll/issue-69114-static-mut-ty.rs

```diff
@@ -7,9 +7,13 @@ static mut BAR_ELIDED: &u8 = &FOO;
 fn main() {
     unsafe {
         println!("{} {}", BAR, BAR_ELIDED);
+        //~^ WARN creating a shared reference to mutable static is discouraged [static_mut_refs]
```

Comment:
Since this test is unrelated, let's fix (or `allow(..)` and mark `FIXME`) this and other similar uses instead.